requirements/base.txt

```diff
@@ -0,0 +1,14 @@
+attrs==20.3.0
+cffi==1.14.4
+cryptography==3.3.1
+importlib-metadata==2.1.1
+more-itertools==8.6.0
+packaging==20.7
+pathlib2==2.3.5
+pluggy==0.13.1
+py==1.9.0
+pyparsing==2.4.7
+pytest==5.4.0
+six==1.15.0
+wcwidth==0.2.5
+zipp==1.2.0
```

Comment:
At some point we had discussed using `>=` rather than `==` so that we may keep following the updates to these components and detect when/if it starts failing with a version to fix our tests quickly, rather than discovering a few years later that we have a huge technical debt